ui/src/app/components/app-router.tsx

```diff
@@ -92,7 +92,10 @@ export const AppRouter = ({children}): React.ReactElement => {
   return <BrowserRouter getUserConfirmation={getUserConfirmation}>{children}</BrowserRouter>;
 };
 
-export const RouteLink = ({path, style = {}, children}): React.ReactElement => <Link style={{...style}} to={path}>{children}</Link>;
+// Most internal routing is done via custom styled Button, not via text, so we only want to use anchor styling
+// if we explicitly set it on the RouteLink
```

Comment:
This changes the link in `not-found` by removing the link underline and color, where I think it is desired.

Please update styles for `not-found` and any other existing uses of RouteLink - except where changing it like this would be an improvement.

Reply:
these all look good now, thanks